hazelcast-client/src/test/java/com/hazelcast/client/map/impl/nearcache/ClientMapNearCacheStalenessTest.java

```diff
@@ -116,7 +116,7 @@ public void testNearCache_notContainsStaleValue_whenUpdatedByMultipleThreads() t
         }
 
         // give some-time to receive possible latest invalidation events
-        sleepSeconds(5);
+        sleepSeconds(30);
```

Comment:
Maybe you can use the `ClientMapInvalidationListener` from the unified Near Cache tests to get rid of this sleep and wait for an invalidation count instead. You can create it like this:
```java
NearCacheInvalidationListener invalidationListener
    = ClientMapInvalidationListener.createInvalidationEventHandler(clientMap)
```
and use it like this:
```java
assertTrueEventually(new AssertTask() {
    @Override
    public void run() throws Exception {
        assertEqualsFormat("Expected %d Near Cache invalidations, but found %d (%s)",
            expectedInvalidations, invalidationListener.getInvalidationCount(), nearCacheStats);
    }
});
invalidationListener.resetInvalidationCount();
```